ArduCopter/control_sport.cpp

```diff
@@ -68,7 +68,7 @@ void Copter::sport_run()
 
     // get pilot desired climb rate
     float target_climb_rate = get_pilot_desired_climb_rate(channel_throttle->get_control_in());
-    target_climb_rate = constrain_float(target_climb_rate, -g.pilot_velocity_z_max, g.pilot_velocity_z_max);
+    target_climb_rate = constrain_float(target_climb_rate, -g2.pilot_velocity_z_max_dn, g.pilot_velocity_z_max_up);
```

Comment:
It's not calling the method.

Reply:
Hmmm, annoyed at myself on this one.  I've fixed it up in the latest commit.